### PR TITLE
CR-1151706 XRT service bundle for collecting analytics data (#7340)

### PR DESCRIPTION
**Pull request by xdavidz**
* CR-1151706 XRT service bundle for collecting analytics data

Signed-off-by: David Zhang <davidzha@xilinx.com>

#### Problem solved by the commit
This is a clean backport. We need this tool for 2022.2 releases for V70.

### CODE REVIEW

**Review by stsoe — APPROVED** [review]

**Comment by dayeh-xilinx** [comment]
retest this please.

**Comment by dayeh-xilinx** [comment]
retest this please.

**Comment by dayeh-xilinx** [comment]
retest this please.

**Review by maxzhen — APPROVED** [review]

File: src/runtime_src/tools/scripts/CMakeLists.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@ set(XRT_LOADER_SCRIPTS
   loader)
```
Comment (rozumx) [inline src/runtime_src/tools/scripts/CMakeLists.txt:11]: Copyright date line 2 needs to be updated.

**Review by rozumx — APPROVED** [review]
Approved.

File: src/runtime_src/tools/scripts/CMakeLists.txt (hunk 1 above)
Comment (xdavidz) [reply src/runtime_src/tools/scripts/CMakeLists.txt:11]: yes, I will update this with next fix.

**Pull request merged**